static/scss/components/buttons.scss

```diff
@@ -35,3 +35,9 @@
     align-items: center;
   }    
 }
+
+.mzp-c-button.has-icon {
```

Comment:
The `.mzp-c-button` does not currently support icons inside itself. This is an override to handle that.